portfolio/src/main/webapp/script.js

```diff
@@ -45,3 +45,63 @@ function addCat() {
   const catPic = document.getElementById("cat-pic");
   catPic.src = cat;
 }
+
+function createCommentElement(comment) {
+  const commentElement = document.createElement("div");
+  commentElement.className = "comment";
+
+  const titleElement = document.createElement("h1");
+  titleElement.innerText = comment[2];
```

Comment:
might be better for the comment data to be in an object rather than array, so that a user can do something like comment.title to get the title rather than have to remember that index [2] is the one with the title....